projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -130,10 +130,11 @@ public function initial_state() {
 		global $publicize;
 
 		return array(
-			'apiRoot'           => esc_url_raw( rest_url() ),
-			'apiNonce'          => wp_create_nonce( 'wp_rest' ),
-			'registrationNonce' => wp_create_nonce( 'jetpack-registration-nonce' ),
-			'connections'       => $publicize->get_all_connections_for_user(),
+			'apiRoot'                          => esc_url_raw( rest_url() ),
+			'apiNonce'                         => wp_create_nonce( 'wp_rest' ),
+			'registrationNonce'                => wp_create_nonce( 'jetpack-registration-nonce' ),
+			'connections'                      => $publicize->get_all_connections_for_user(),
+			'jetpackSocialConnectionsAdminUrl' => $publicize->publicize_connections_url( 'jetpack-social-connections-admin-page' ),
```

Comment:
I know it shouldn't be an issue, but just in case, should we espape this URL like `apiRoot`? Similarly should we be sanitising the `connections` in some way? I don't know if everything has already been sanitised and escaped by this point.